src/Probabilities.cpp

```diff
@@ -554,6 +557,7 @@ Probabilities::initialize_single_probs()
 	m[pFuncAttrInline] = 50;
 	m[pFuncAttrProb] = 30;
 	m[pTypeAttrProb] = 50;
+	m[pTransAtomicProb] = 50;
```

Comment:
This is very high, I would set it to 3% or so?

Reply:
Done.